symfony/framework-bundle/7.2/config/packages/framework.yaml

```diff
@@ -8,6 +8,11 @@ framework:
     #esi: true
     #fragments: true
 
+    # Enable stateless CSRF protection for forms and logins/logouts
+    form: { csrf_protection: { token_id: submit } }
+    csrf_protection:
+        stateless_token_ids: [submit, authenticate, logout]
```

Comment:
It might be nice to either have some explanations about what those values are about and where they come from so that one knows what to do or not with this option, or linking to a docs page telling that (once we have one).

Reply:
Isn't it what L11 is about? Getting into more details might be lengthy and not a fit for this file?